### DISCUSSION TITLE
Suggestion: search on the state

### DISCUSSION THREAD

**Discussion by jcarlos** [reduxjs/redux-devtools#1093] [Ideas]
If I press cmd+f I cannot search the state tree to quickly find a key / value. The only way to search is to choose "raw" and then copy it over to a text editor and do a search inside the text editor. Would be awesome if I could hit cmd+f / ctrl + f and search inside the state values / keys.

**Comment by jonahlibrach** [comment]
I'd benefit from this greatly – and it seems like it wouldn't be too tough to implement if it were also easy to get the development version up an running

**Comment by IMMihaylov** [comment]
+1

**Comment by unreal777** [comment]
its currently working for the redux in separate panel, but using it directly from devTools doesnt allow search, and this would be really usefull.

**Reply by laurentlahmy** [reply]
I don't think it works in a seperate panel

**Comment by mirastroie** [comment]
Can you please reconsider implementing this? I have to copy-paste the state in an editor to search for something while I use Redux directly from devTools 😞

**Comment by laurentlahmy** [comment]
would be useful 🙏
